### ISSUE TITLE
[Feature] Option to compare text with original selection

### ISSUE THREAD

**Issue by Seviks** [theJayTea/WritingTools#25]
While immediate text replacement is often preferred, there are instances where comparing the new text to the existing one is beneficial. 

Could you consider implementing a switch for appending or replacing text in the pop-up window? If set to "append text," the text would be inserted below the selected content. Naturally, there may be more effective methods to implement such a feature.

**Comment by theJayTea** [comment]
That’s an interesting proposition. Although, have you considered this:
The most effective way to compare (for me) is to keep cycling over `ctrl+z`, followed by `ctrl+y`.
The changes made (for instance, with proofread) will be pretty obvious this way :)

**Comment by Seviks** [comment]
Yes, this is how I/we do it today. However, if it is used to alter several sentences, it can become difficult to notice the changes. In any case, I don't want to request it only for myself; perhaps we can gather more pro and con arguments.

**Comment by minzdrav** [comment]
Hi folks,
+1 for changes preview.
It can be useful if I don't want to change the text. For example, I want to ask AI to explain the code that's been selected in Visual Studio (yes, I can use a VS plugin for that) or summarize history from chat.

**Comment by vfxturjo** [comment]
Please add this feature.

It would be very helpful for people if there were options to:
- Directly apply changes
- Append changed texts below the original
- **Send to chat result window**: allows users to see input text and output text for comparison, with the accept/reject button and further modification through follow-up prompts.

the options could be cycled/toggled through a button/toggle placed on either the red or the purple area marked in the image below:
![Image](https://github.com/user-attachments/assets/285f531e-b68f-4623-8125-796acf5d9f5a)

**Comment by theJayTea** [comment]
Hello :)

This is something I'll work on shortly after we release customizable buttons, so hopefully in the v8 release.

@vfxturjo thank you for the UX idea, I'll keep that in mind as I brainstorm the best approach to implement this from a UX perspective.
